build/launcher.Dockerfile

```
@@ -28,6 +28,8 @@ RUN CGO_ENABLED=0 \
 
 FROM debian:bookworm-slim
 
+ARG CONTAINERLAB_VERSION="0.0.0~14789vxlan"
```

let's maybe default to the tested version instead of a dev one?
```suggestion
ARG CONTAINERLAB_VERSION="0.45"
```

---

ah yes :) I was just gunna not merge this till next clab release and updating this. but can merge it now w/ 0.45.* if ya think we should just do it now?

---

done, but some issues w/ clab VxLAN tools stuff, so will hold off on merging this just yet. tl;dr [here](https://github.com/srl-labs/containerlab/blob/d38bdf37a6504a8bf47c71275f06813f1ef21176/links/link_vxlan.go#L54) looks like we want (at least for this use case, and since I dont know enough of all the vxlan stuff that happened lately I dont wanna pr and break other things!) `vx-%s` rather than what ends up looking like `ve-host_XYZ` and/or the alias like `vx-NODENAME-INTF` maybe should change to be ve-host_XYZ?

---

tracking in https://github.com/srl-labs/containerlab/pull/1625

---

tested w/ build off that branch and looks good! can update version and merge tomorrow or whenever ya cut next release!